modules/azure/function_app_linux_managed_identity/main.tf

```diff
@@ -181,6 +165,7 @@ resource "azurerm_monitor_diagnostic_setting" "diagnostic_setting" {
 
     content {
       category = enabled_log.value
+      enabled  = true
```

Comment:
You still forgot to revert this change, this is part of the change to enabled_log